apis/DefaultApi.ts

```diff
@@ -73,7 +73,7 @@ export class DefaultApiRequestFactory extends BaseAPIRequestFactory {
         requestContext.setHeaderParam("Accept", "application/json, */*;q=0.8")
 
         // Always add the One Signal telemetry to the request.
-        requestContext.setHeaderParam("OS-Usage-Data", "kind=sdk, sdk-name=onesignal-typescript, version=5.0.0-alpha-01");
+        requestContext.setHeaderParam("OS-Usage-Data", "kind=sdk, sdk-name=onesignal-typescript, version=5.0.1");
```

Comment:
Maybe something to consider for a future update. Shouldn't these headers match the version in the `package.json`? Right now it seems to be tracking with API spec version, which is less granular?

Reply:
> Maybe something to consider for a future update.
Agreed, we can reconsider. This is internal only for our own usage metrics.